sockeye/output_handler.py

```diff
@@ -140,3 +115,29 @@ def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.i
                              t_input.tokens,
                              t_output.tokens,
                              self.threshold)
+
+
+def get_output_handler(output_type: str,
```

Comment:
No major motivation on my part, I'll revert it back.  If we used the type as a return hint we'll get a not declared error, but I'm assuming you're already aware of this given that you're using a string here.  

Any motivation other than consistency for the forward declaration?